core/baseline-core-v1.0-psd01.md

```diff
@@ -4746,13 +4816,62 @@ Bi- or Multi-directional BPI services in the context of BPI interoperability MUS
 * *Validity requirements of an operation's data properties*
 * *Operation execution requirements* 
 
+[[R139]](#r139) Testability: 
+
+Preconditions:
+
+* Two or more BPI systems are set up for bi- or multi-directional BPI services.
+* All BPI systems involved are configured for interoperability.
+* The BPI Subjects and Participants are properly registered in their respective systems.
+
+Test Steps:
+
+1. Create a State Synchronization and Advancement Predicate from one BPI system.
+2. Update the State Synchronization and Advancement Predicate from another BPI system.
+3. Launch an IVSM (Interoperable Verification Smart Master) from one of the BPI systems.
```

Comment:
IVSM = Interoperability Virtual State Machine ... no need to spell it out since it is defined already above

Reply:
It would be nice if odd terms and acronyms could link to where they are defined.